src/ray/util/logging.cc

```diff
@@ -67,6 +67,7 @@ typedef ray::CerrLog LoggingProvider;
 
 RayLogLevel RayLog::severity_threshold_ = RayLogLevel::INFO;
 std::string RayLog::app_name_ = "";
+bool RayLog::is_stored_ = false;
```

Comment:
Let's define these default values in the header file.

Reply:
Static member can be initialized in the header file only if it is constant, log_dir_(is_stored_) could be changed.